app/services/stripe-details.service.js

```diff
@@ -0,0 +1,28 @@
+const { updateBankAccount, errorCodes } = require('./clients/stripe/stripe.client')
```

Comment:
`errorCodes` doesn't appear to be a method on the `stripe.client`

Reply:
good spot, I was originally trying to see if I could somehow get a hold of Stripe error codes for bank account details without hard coding the values in my controller. This should have been removed